SPECS/clamav/clamav.spec

```diff
@@ -136,6 +136,9 @@ fi
 %dir %attr(-,clamav,clamav) %{_sharedstatedir}/clamav
 
 %changelog
+* Fri Mar 07 2025 Kavya Sree Kaitepalli <kkaitepalli@microsoft.com> -1.0.7-2
```

Comment:
empty space. missing. please run SPEC sanitization tooling, if not done, to catch such things